google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BigtableCloudMonitoringExporter.java

```diff
@@ -290,22 +287,17 @@ private CompletableResultCode exportApplicationResourceMetrics(
 
     // Construct the request. The project id will be the project id of the detected monitored
     // resource.
-    ApiFuture<Empty> gceOrGkeFuture;
+    ApiFuture<List<Empty>> gceOrGkeFuture;
     CompletableResultCode exportCode = new CompletableResultCode();
     try {
       ProjectName projectName =
           ProjectName.of(applicationResource.getLabelsOrThrow(APPLICATION_RESOURCE_PROJECT_ID));
-      CreateTimeSeriesRequest request =
-          CreateTimeSeriesRequest.newBuilder()
-              .setName(projectName.toString())
-              .addAllTimeSeries(timeSeries)
-              .build();
 
-      gceOrGkeFuture = this.client.createServiceTimeSeriesCallable().futureCall(request);
+      gceOrGkeFuture = exportTimeSeries(projectName.toString(), timeSeries);
```

Comment:
Why toString? I think it would make sense to keep it typesafe as long as possible